src/wordpress.js

```diff
@@ -82,9 +89,7 @@ async function fetchAll({ numPages, endPoint, type }) {
 }
 
 async function fetchData(type, endPoint) {
-  const baseURL = process.env.WORDPRESS_BASE_URL || DEFAULT_WORDPRESS_BASE_URL;
-
-  const url = `${baseURL}${endPoint}`;
+  const url = `${WORDPRESS_BASE_URL}${endPoint}`;
```

Comment:
Might be good to validate that endPoint always starts with a '/' and throw if not.

Reply:
done